pyproject.toml

```diff
@@ -3,27 +3,37 @@ requires = ["pdm-backend"]
 build-backend = "pdm.backend"
 
 [tool.pdm.build]
-includes = [
-    "stac_model",
-]
+includes = ["stac_model"]
 
 [project]
 authors = [
-    {name = "Ryan Avery", email = "ryan@wherobots.com"},
-    {name = "Francis Charette-Migneault", email = "francis.charette-migneault@crim.ca"},
+  { name = "Ryan Avery", email = "ryan@wherobots.com" },
+  { name = "Francis Charette-Migneault", email = "francis.charette-migneault@crim.ca" },
 ]
-license = {text = "Apache Software License 2.0"}
+license = { text = "Apache Software License 2.0" }
 requires-python = "<4.0,>=3.10"
 dependencies = [
-    "typer<1.0.0,>=0.9.0",
-    "rich>=13.7.0,<15.0.0",
-    "pydantic<3.0.0,>=2.6.3",
-    "pydantic-core<3,>=2",
-    "pystac<2.0.0,>=1.9.0",
-    "shapely<3,>=2",
-    "jsonschema<5.0.0,>=4.21.1",
-    "pip>=25.0.0",
+  "typer<1.0.0,>=0.9.0",
+  "rich>=13.7.0,<15.0.0",
+  "pydantic<3.0.0,>=2.6.3",
+  "pydantic-core<3,>=2",
+  "pystac<2.0.0,>=1.9.0",
+  "shapely<3,>=2",
+  "jsonschema<5.0.0,>=4.21.1",
+  "pip>=25.0.0",
+]
+optional-dependencies.torch = [
+  "torch==2.8.0; python_version>='3.11'",
+  "torchgeo; python_version>='3.11'",
+  "torchvision>=0.21,<1; python_version>='3.11'"
+]
+optional-dependencies.torch-cu126 = [
+  "pytorch-triton; python_version>='3.11'",
+  "torch==2.8.0; python_version>='3.11'",
+  "torchgeo; python_version>='3.11'",
+  "torchvision>=0.21,<1; python_version>='3.11'",
 ]
+
 # important: leave the name and version together for bump resolution
 name = "stac-model"
 version = "0.3.0"
```

Comment:
@fmigneault thinking we should bump to 0.4.0 in a follow up PR and make a release wdyt?

Reply:
Yes. Already planned. As soon as the PR is integrated, it will be bumped along with MLM spec.